enterprise/server/api/api_server.go

```diff
@@ -319,6 +323,13 @@ func (s *APIServer) GetLog(ctx context.Context, req *apipb.GetLogRequest) (*apip
 	}, nil
 }
 
+func (gfs *GetFileServer) Write(data []byte) (int, error) {
```

Comment:
ubernit: you could call the struct getFileWriter (it doesn't need to be exported) and you can just define it and the Write() method on it right above GetFile since that's the only place it's likely to be used.

Reply:
thanks, still getting used to conventions.